bench/internals.rs

```diff
@@ -26,12 +26,12 @@ lazy_static! {
     static ref CONTRACT_CLASS: ContractClass = ContractClass::from_path(
         "starknet_programs/account_without_validation.json",
     ).unwrap();
-    static ref CLASS_HASH: Felt252 = compute_deprecated_class_hash(&CONTRACT_CLASS).unwrap();
-    static ref CLASS_HASH_BYTES: [u8; 32] = CLASS_HASH.clone().to_be_bytes();
+    static ref CLASS_HASH_FELT: Felt252 = compute_deprecated_class_hash(&CONTRACT_CLASS).unwrap();
+    static ref CLASS_HASH: ClassHash = ClassHash(CLASS_HASH_FELT.clone().to_be_bytes());
```

Comment:
The clone should not be necessary.

Reply:
Thanks! You are right, also changed on lib.rs